relay-server/src/utils/sizes.rs

```diff
@@ -42,6 +40,11 @@ pub fn check_envelope_size_limits(config: &Config, envelope: &Envelope) -> bool
             ItemType::Metrics => (),
             ItemType::MetricBuckets => (),
             ItemType::ClientReport => client_reports_size += item.len(),
+            ItemType::ProfilingSession | ItemType::ProfilingTrace | ItemType::Profile => {
+                if item.len() > config.max_profile_size() {
```

Comment:
This now means you can have multiple profiles per envelope adding up to whatever the maximum envelope size is. This is fine with me, just checking that this is intentional?

Reply:
Yes, this is intentional. I'd like to let clients submit many profiles per envelope.